src/paper/tasks.py

```diff
@@ -1267,3 +1267,157 @@ def pull_openalex_author_works(user_id, openalex_id):
         filters=[NEW],
     )
     return True
+
+
+@periodic_task(
+    run_every=crontab(minute=0, hour=0), priority=3, queue=QUEUE_PULL_PAPERS
```

Comment:
I might change this from midnight to sometime in the early morning. Going to take a look at avg. db load by time of day to get a sense of when would be best.

Reply:
Great thinking